README.md

```diff
@@ -1,6 +1,43 @@
 # tap-google-analytics
 
-This tap is currently under development, and incomplete.
+This is a [Singer](https://singer.io) tap that produces JSON-formatted data following the [Singer spec](https://github.com/singer-io/getting-started/blob/master/SPEC.md).
+
+How to use it:
+- `tap-google-analytics` works together with any other [Singer Target](https://singer.io) to move data from Google Analytics API to any target destination.
+- Extracts the following pre-made and any self-made reports from [Google Analytics Reporting API](https://developers.google.com/analytics/devguides/reporting/core/v4/rest/v4/reports/batchGet):
```

Comment:
Could you mention the field selection for pre-made reports must follow google's exclusion rule, and point users to the Metrics & Dimensions page https://ga-dev-tools.web.app/dimensions-metrics-explorer/. These rules are handled in the Stitch UI by dynamically marking fields as unsupported as you select a field, but will need to be handled by the user if they wish to run the tap directly.